config.yml.example

```diff
@@ -1,8 +1,28 @@
+# This is the configuration file for Icinga DB.
+
 database:
   host: icingadb
   port: 3306
   database: icingadb
   user: icingadb
   password: icingadb
+
 redis:
   address: redis:6380
+
+logging:
+  # Default logging level. Can be set to 'fatal', 'error', 'warning', 'info' or 'debug'.
+  # If not set, defaults to 'info'.
+  level:
+
+  # Map of component-logging level pairs to define a different log level than the default value for each component.
+  options:
+    database:
+    redis:
+    heartbeat:
+    high-availability:
+    config-sync:
+    history-sync:
+    runtime-updates:
+    overdue-sync:
+    dump-signals:
```

Comment:
I miss example levels here. (Only a complete example is a good example.)

Reply:
Example levels are already document above.

Reply:
And in the docs. And all of the other options are also documented in the docs. Nevertheless there are example values here like icingadb or 3306.

Reply:
This file should be self-explanatory or it will be useless. So I don't really get your point.

Reply:
I agree and my point is to make it a bit more self-explanatory, i.e. `s/level:/level: info/`, etc..

Reply:
That would be counterproductive as it would override our built-in default mechanics.

Reply:
... if users just C&P the file?

Reply:
I think this is a use case of this file. Of course, we should update the connection configuration to sane defaults prior to the release.

Reply:
Damn.